Source/Painting/SvgDeferredPaintServerFactory.cs

```diff
@@ -0,0 +1,19 @@
+using System.ComponentModel;
+using System.Globalization;
+
+namespace Svg
+{
+    internal class SvgDeferredPaintServerFactory : SvgPaintServerFactory
```

Comment:
I create this Factory for `xlink:href` because `SvgPaintServerFactory` cannot make an accurate decision.

Reply:
Can you please add a bit of explanation to the factory or the `ConvertFrom` method? It took some time for me to understand why it is needed...